src/styles/globals.css

```diff
@@ -10,5 +12,9 @@
   --color-alternative-bright: #fafafa;
   --color-primary: #c084fc;
   --color-primary-dark: #a855f7;
+  --color-mineShaft: #484848;
```

Comment:
this name is so specific as to not be a variable. Please name it for a role (e.g. `text`) or an approximate color (like `gray`)

Reply:
I get why the Figma would use it, because they're designing a screen that doesn't have to be maintained. In code we should not name literal colors. If there were more time I'd say to work with the designers to understand how they _use_ "mine shaft" and ask them to give it a semantic name. But I won't block on this.